packages/devtools_app/lib/src/charts/treemap.dart

```diff
@@ -382,44 +357,39 @@ class Treemap extends StatelessWidget {
   /// ----------------------------
   /// ```
   Widget buildTreemap(BuildContext context) {
-    if (levelsVisible > 0 && rootNode.children.isNotEmpty) {
+    if (rootNode.children.isNotEmpty) {
       return Padding(
         padding: const EdgeInsets.all(1.0),
-        child: Column(
-          children: [
-            if (height > minHeightToDisplayTitleText) buildTitleText(context),
-            Expanded(
-              child: Treemap.fromNodes(
-                nodes: rootNode.children,
-                levelsVisible: levelsVisible,
-                isOutermostLevel: isOutermostLevel,
-                onRootChangedCallback: onRootChangedCallback,
-                height: height,
+        child: buildSelectable(
+          selectable: !isOutermostLevel,
+          child: Column(
+            children: [
+              if (height > minHeightToDisplayTitleText) buildTitleText(context),
+              Expanded(
+                child: Treemap.fromNodes(
+                  nodes: rootNode.children,
+                  levelsVisible: levelsVisible,
+                  isOutermostLevel: isOutermostLevel,
```

Comment:
will isOutermostLevel ever be true if we are buildling a treemap `fromNodes`? if not, can we remove this parameter or make it default to false